distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -90,3 +90,7 @@ MOVE;$OPENHAB_USERDATA/config/org/eclipse/smarthome/storage/json.config;$OPENHAB
 REPLACE;org.eclipse.smarthome.storage.json;org.openhab.storage.json;$OPENHAB_USERDATA/config/org/openhab/storage/json.config
 MOVE;$OPENHAB_USERDATA/config/org/eclipse/smarthome/storage/mapdb.config;$OPENHAB_USERDATA/config/org/openhab/storage/mapdb.config
 REPLACE;org.eclipse.smarthome.storage.mapdb;org.openhab.storage.mapdb;$OPENHAB_USERDATA/config/org/openhab/storage/json.config
+
+DELETE;$OPENHAB_USERDATA/config/org/openhab/classicui.config
```

Comment:
IIRC we have to add `DELETE` and `DELETEDIR` to the [linuxpkg](https://github.com/openhab/openhab-linuxpkg/blob/29ca018ff62e71ac4646f5c270bc4e6953e3f741/resources/control-runtime/postinst#L129). Have we?

Reply:
Yes seems like it. @BClark09 first wanted to test these before adding them  (see https://github.com/openhab/openhab-linuxpkg/pull/152).